### PR TITLE
fix an issue w/ references in dartdoc not being hyperlinked

### PR DESCRIPTION
**Pull request by devoncarew**
Fix https://github.com/dart-lang/dartdoc/issues/1152; this was fixed by upgrading to the latest markdown.

@keertip, @pq

### CODE REVIEW

**Comment by pq** [comment]
Nice.  LGTM!

**Comment by keertip** [comment]
Great! lgtm!

File: testing/test_package_docs/ex/ex-library.html [hunk 1]
```diff
@@ -92,7 +92,7 @@ <h1 class="title">
   <div class="col-xs-12 col-sm-9 col-md-8 main-content">
 
     <section class="desc markdown">
-      <p>a library. testing string escaping: <code>var s = 'a string'</code> &lt;cool&gt;</p>
+      <p>a library. testing string escaping: <code>var s = 'a string'</code> <cool></cool></p>
```
Comment (Hixie) [inline testing/test_package_docs/ex/ex-library.html]: this seems wrong?

File: testing/test_package_docs/index.html [hunk 2]
```diff
@@ -84,11 +84,11 @@ <h1 class="title">
 <p>This is an amazing package.</p>
 <p>For example, it:</p><ul><li>Is very fast</li><li>Has zero bugs</li><li>Is free</li></ul>
 <p>It also has some awesome code</p>
-<pre class="dart prettyprint"><code>void main() {
+<pre class="prettyprint language-dart"><code class="language-dart">void main() {
   // in Dart!
 }
 </code></pre>
-<pre class="yaml prettyprint"><code>and_yaml:
+<pre class="prettyprint language-dart"><code class="language-yaml">and_yaml:
```
Comment (Hixie) [inline testing/test_package_docs/index.html]: why not yaml?

File: testing/test_package_docs/ex/ex-library.html (hunk 1 above)
Comment (devoncarew) [inline testing/test_package_docs/ex/ex-library.html]: The newer markdown library handles passing through html embedded in the markdown.

File: testing/test_package_docs/index.html (hunk 2 above)
Comment (devoncarew) [inline testing/test_package_docs/index.html]: Hmm, let me look into this.

**Pull request merged**